src/BankAccountApp/CreateAccountsFromFile.java

```diff
@@ -0,0 +1,56 @@
+package BankAccountApp;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+// read a CSV File then create new accounts based on that data
+
+class CreateAccountsFromFile {
+
+
+    ArrayList<Account> createAccountsBatch() {
+
+        ArrayList<Account> accounts = new ArrayList<Account>();
+
+        try {
+            File file = new File("/Users/kelly.powell/Downloads/Bank_Application/NewBankAccounts.csv");
+            Scanner scanner = new Scanner(file);
+            label:
+            while (scanner.hasNextLine()) {
+                String readLine = scanner.nextLine();
+                String[] splitLine = readLine.split(",");
+                String name = splitLine[0];
+                String sSN = splitLine[1];
+                String accountType = splitLine[2];
+                Double initDeposit = Double.parseDouble(splitLine[3]);
+
+                switch (accountType) {
+                    case "BankAccountApp.Savings":
+                        Savings newSavings = new Savings(name, sSN, initDeposit);
+                        newSavings.showInfo();
+                        accounts.add(newSavings);
+
+                        break;
+                    case "BankAccountApp.Checking":
+                        Checking newChecking = new Checking(name, sSN, initDeposit);
+                        newChecking.showInfo();
+                        accounts.add(newChecking);
+
+                        break;
+                    default:
+                        System.out.println("Incorrect field for account type! Please check file.");
+                        break label;
+                }
+
+            }
+        } catch (FileNotFoundException e) {
```

Comment:
This only occurs when created instantiating a new `File` so there's not possibility of this if we're reading through the file in the `while` loop

Reply:
I've tried a few different ways of moving/removing the try/catch block and I keep running into errors. Maybe you can take a quick look at this with me tomorrow? Thank you.